Character.h

```diff
@@ -19,4 +19,15 @@ class Character {
 	static Character parseUnit(const std::string& FileName);
 };
 
+class Player : public Character {
+	int xp;
+	int MaxHp;
+public:
+	Player(const std::string name, const int hp, int dmg, const int xp = 0) : Character(name, hp, dmg), xp(xp), MaxHp(hp) {}
+	int getXp() const;
+	void setXp();
+	int LevelUp();
```

Comment:
Ezek ne legyenek publikusak, ne az osztály használójának kelljen foglalkoznia az xp-vel, azt kezelje automatikusan az osztály a harc lebonyolításánál.

Reply:
Köszönöm, ezt is javítottam!